cohort-engine/src/main/java/com/ibm/cohort/engine/BooleanEvaluationCallback.java

```diff
@@ -32,6 +32,7 @@ public void onContextBegin(String contextId) {
 
 	@Override
 	public void onContextComplete(String contextId) {
+		logger.info(contextId + "complete");
```

Comment:
Adding a space before `complete` will make the log line a little more readable

Reply:
Fixed